app/services/identifier_service.rb

```diff
@@ -21,7 +21,7 @@ def self.get_ark_result(ark:)
 
   def self.url_for(resource)
     return Rails.application.routes.url_helpers.solr_document_url(resource, host: Figgy.default_url_options[:host]) if resource.try(:source_metadata_identifier).blank?
-    return "https://catalog.princeton.edu/catalog/#{resource.source_metadata_identifier.first}#view" if PulMetadataServices::Client.bibdata?(resource.source_metadata_identifier.first)
```

Comment:
This will change where all of our ARKs point, which seems like an unintended side effect.

Reply:
Will this mess up existing ARKs?

Reply:
I don't really understand why they point there.

Reply:
It will mess up any ARKs that are updated, yeah. They point here so that when you click the ARK it jumps to the viewer if possible.

Reply:
Okay I pushed a better idea.